.travis.yml

```diff
@@ -10,16 +10,17 @@ python:
  - "2.7"
  - "2.6"
 
+install:
+ - pip install flake8
+
 script:
  # Simple test runs
  - ./local-install.sh
  - ./curl-install.sh
 
-after_script:
- # Static analysis: for info, won't fail the build
+ # Static analysis
  - pip install pyflakes pycodestyle
- - pyflakes .
- - pycodestyle --statistics --count .
+ - flake8 .
```

Comment:
You known that flake8 uses pyflakes with a bit permissive config, right?
Oh, and how about using `black`?
Also idea: there's a nice project pre-commit.com, which enables users to have local hook in git as well as repeat that in CI. It supports running all linters above (and more) and I can help configuring it if needed.

Reply:
I switched to flake8 because pyflakes doesn't respect the `# noqa` comments. But now flake8 doesn't install for all Python versions. But we don't need to run it on all, just a recent one. Will update.

Black could be good too, but I'd probably wait until it's at least in beta, as things are still changing there.

Reply:
we can add extra allowed to fail job for black